study.md

```diff
@@ -46,14 +49,30 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+<!-- It signifies change to the object and not just a point to it.
+The green-checked response explains how I understand it.
+http://stackoverflow.com/questions/7179016/what-is-the-purpose-of-and-at-the-end-of-method-names-->
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+
+# Can instantiate a class in Ruby by typing the following in
+#the terminal (irb to get Ruby going in Terminal)
+class Greeter
+  def initialize(name = "World")
+    @name = name
+  end
+  def say_hi
+    puts "Hi #{@name}!"
+  end
+  def say_bye
+    puts "Bye #{@name}, come back soon."
+  end
+end
+
```

Comment:
You are defining a class, we wanted you to make a new instance of a class like so:

`my_instance = MyClass.new`